### ISSUE TITLE
little mistake in the repo example: change logs_root to logs_dir

### ISSUE THREAD

**Issue by mjmaher987** [omarfoq/FedEM#9]
there is no argument with name logs_root. but when you change it to logs_dir it's ok. example:
! python3  FedEM/run_experiment.py cifar10 FedAvg \
    --n_learners 1 \
    --n_rounds 200 \
    --bz 128 \
    --lr 0.01 \
    --lr_scheduler multi_step \
    --log_freq 5 \
    --device cuda \
    --optimizer sgd \
    --seed 1234 \
    --logs_dir /content \
    --verbose 1
